package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "abagnale",
+  "version": "1.0.0",
+  "description": "Forge unique IDs for Refract data structure elements",
+  "main": "index.js",
+  "scripts": {
+    "test": "peasant test",
+    "cover": "peasant cover",
+    "lint": "peasant lint",
+    "peasant": "peasant",
+    "ci": "peasant -s lint test build",
+    "prepublish": "npm run ci"
+  },
+  "author": "Apiary, Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apiaryio/abagnale.git"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "glob": "^7.0.0",
+    "peasant": "^0.5.2"
```

Comment:
The API Blueprint team knows about it, but maybe not all of engineering. I can present it but at the moment it's quite basic and doesn't work with e.g. React. It also now uses an outdated Babel, so one of these days soon I should update Peasant.